.rspec

```diff
@@ -0,0 +1,2 @@
+--color
+--format documentation
```

Comment:
Should this file have been checked in? Generally we have this in the `.gitignore`, so people can put their own customizations in there.